### DISCUSSION TITLE
v0.7.5

### DISCUSSION THREAD

**Discussion by levskaya** [google/flax#3445] [General]
## What's Changed
* Add method-to-model section to Haiku migration guide by @IvyZX in https://github.com/google/flax/pull/3277
* updated haiku guide with new JAX RNG api by @chiamp in https://github.com/google/flax/pull/3343
* changed resnet v1 to v1.5 by @chiamp in https://github.com/google/flax/pull/3344
* updated flax_basics by @chiamp in https://github.com/google/flax/pull/3342
* Add `find` methods and magic methods for Cursor API by @chiamp in https://github.com/google/flax/pull/3306
* fix DeprecationWarnings by @chiamp in https://github.com/google/flax/pull/3352
* Make checkpoint guide path absolute by @IvyZX in https://github.com/google/flax/pull/3358
* use jax.Array type for rng keys by @chiamp in https://github.com/google/flax/pull/3354
* Disable pyink by @cgarciae in https://github.com/google/flax/pull/3356
* removed DeprecationWarning filter by @chiamp in https://github.com/google/flax/pull/3359
* Don't propagate default args in Tabulate by @cgarciae in https://github.com/google/flax/pull/3357
* added spectral norm by @chiamp in https://github.com/google/flax/pull/3335
* fixed bind-unbind bug by @chiamp in https://github.com/google/flax/pull/3365
* Add MaxText open source LLM to index.rst by @8bitmp3 in https://github.com/google/flax/pull/3368
* fixed typo by @chiamp in https://github.com/google/flax/pull/3367
* Add fp8 custom op and unit test by @wenscarl in https://github.com/google/flax/pull/3322
* fixed scope typing by @chiamp in https://github.com/google/flax/pull/3371
* Trailing whitespace fixes. by @levskaya in https://github.com/google/flax/pull/3373
* Make fp8 ops use explicit broadcasting. by @levskaya in https://github.com/google/flax/pull/3374
* Conditonal contraints for numpy and clu by @cgarciae in https://github.com/google/flax/pull/3394
* add truncated_normal to initializers by @levskaya in https://github.com/google/flax/pull/3401
* fix HEAD by @chiamp in https://github.com/google/flax/pull/3404
* updated imagenet readme by @chiamp in https://github.com/google/flax/pull/3383
* Add Flax FAQ - how to search, how to take the derivative w.r.t. a hidden layer, remat_scan vs scan(remat), recommended training libraries/metrics by @8bitmp3 in https://github.com/google/flax/pull/3267
* added rmsnorm to api docs by @chiamp in https://github.com/google/flax/pull/3406
* updated docs by @chiamp in https://github.com/google/flax/pull/3370
* fix HEAD by @chiamp in https://github.com/google/flax/pull/3408
* added weightnorm layer by @chiamp in https://github.com/google/flax/pull/3405
* added attention refactor to changelog by @chiamp in https://github.com/google/flax/pull/3412
* added dropout arg to `MultiHeadDotProductAttention` by @chiamp in https://github.com/google/flax/pull/3384
* fix spectralnorm layer by @chiamp in https://github.com/google/flax/pull/3403
* remove pdf target by @cgarciae in https://github.com/google/flax/pull/3415
* added import precommit hook by @chiamp in https://github.com/google/flax/pull/3410
* fixed GRU docstring by @chiamp in https://github.com/google/flax/pull/3419
* Replaces pjit with jit in spmd.py by @copybara-service in https://github.com/google/flax/pull/3421
* Ignore transient chex deprecationwarning. by @levskaya in https://github.com/google/flax/pull/3427
* Remove transformers dependency from docs by @cgarciae in https://github.com/google/flax/pull/3431
* updated pytorch upgrade guide by @chiamp in https://github.com/google/flax/pull/3432
* Simplify abstract rng creation in param shape check. by @levskaya in https://github.com/google/flax/pull/3429
* [NVIDIA] Update the FP8 support by @kaixih in https://github.com/google/flax/pull/3435
* fixed mypy errors at HEAD by @chiamp in https://github.com/google/flax/pull/3440
* added `has_improved` field to EarlyStopping by @chiamp in https://github.com/google/flax/pull/3385
* fully deprecated old RNN api by @chiamp in https://github.com/google/flax/pull/3425
* updated lm1b example with jit by @chiamp in https://github.com/google/flax/pull/3302
* added MGU class by @chiamp in https://github.com/google/flax/pull/3418
* Make Flax Basics visible by @8bitmp3 in https://github.com/google/flax/pull/3443
* update for release v0.7.5 by @levskaya in https://github.com/google/flax/pull/3444

## New Contributors
* @wenscarl made their first contribution in https://github.com/google/flax/pull/3322
* @kaixih made their first contribution in https://github.com/google/flax/pull/3435

**Full Changelog**: https://github.com/google/flax/compare/v0.7.4...v0.7.5

<hr /><em>This discussion was created from the release <a href='https://github.com/google/flax/releases/tag/v0.7.5'>v0.7.5</a>.</em>
